### PR TITLE
bug/1382_carto_enterprise_individual_accounts

### PR DESCRIPTION
**Pull request by frbattid**
* Fixes issue #1382 
* e2e tests done by @emepetres, who owns a personal Carto account with API key :)

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -8,3 +8,4 @@
 - [cygnus-ngsi][hardening] Add raw bytes to NGSIEvent (#1345)
 - [cygnus-ngsi][hardening] Add cache to NGSIMySQLSink (#130)
 - [cygnus][hardening] Fix "FIWARE" name in license headers (#1369)
+- [cygnus-ngsi][bug] Differentiate among enterprise and individual accounts (#1382)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: Maybe better

[cygnus-ngsi][bug] Differentiate among enterprise and individual Carto accounts (#1382)

?

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/cartodb/CartoDBBackendImpl.java [hunk 2]
```diff
@@ -31,6 +31,7 @@
 public class CartoDBBackendImpl extends HttpBackend implements CartoDBBackend {
     
     private final String apiKey;
+    private final boolean isPersonal;
```
Comment (fgalan) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/cartodb/CartoDBBackendImpl.java]: isPersonal -> isPersonalAccount ?

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (frbattid) [inline CHANGES_NEXT_RELEASE]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1393/commits/f2d7e43266253d141ea7e0f83ade64436c238e59

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/cartodb/CartoDBBackendImpl.java (hunk 2 above)
Comment (frbattid) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/cartodb/CartoDBBackendImpl.java]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1393/commits/f2d7e43266253d141ea7e0f83ade64436c238e59

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/cartodb/CartoDBBackendImpl.java [hunk 3]
```diff
@@ -39,17 +40,23 @@
      * @param port
      * @param ssl
      * @param apiKey
+     * @param isPersonal
```
Comment (fgalan) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/cartodb/CartoDBBackendImpl.java]: isPersonal -> isPersonalAccount

Comment (frbattid) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/cartodb/CartoDBBackendImpl.java]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1393/commits/642cc873e21caa7b431ae9f3a59ca3a661e27039

**Comment by fgalan** [comment]
LGTM

**Pull request merged**